app/views/user/shares/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= bootstrap_form_with(model: share, scope: 'share', local: true) do |f| %>
+  <% if share.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(share.errors.count, "error") %> prohibited this share from being saved:</h2>
+
+      <ul>
+      <% share.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= f.select(
+    :birth_record_id, 
+    current_user.birth_records.map{ |r| [r.short_name, r.id] },
+    selected: share.birth_record_id) 
+  %>
+
+  <%= f.select(
```

Comment:
what is this was an autosuggest fields instead - so the user types what who they want to send to, and we suggest matches to select from. These are part of boostrap_form.

Reply:
sure, I just put in the least styling to get this up and running

Reply:
I don't see an autosuggest in the bootstrap_form documentation, are you thinking of an additional gem?

Reply:
Might be. We can add on the next PR then

I suspect having the drop down here for which birth record is confusing. How about we only let them share the one they chose and passed as a get param.

Reply:
This could even be a form renderd on the birthrecord/show, and be a remote: true. IT would mean way less getting lost in all the pages of the app.